src/dealer/DealerWallet.ts

```diff
@@ -0,0 +1,151 @@
+import { accountDealerFtxPath, liabilitiesDealerFtxPath } from "../ledger/ledger"
+import { MainBook } from "../mongodb"
+import { Result } from "./Result"
+import { OnChainMixin } from "../OnChain"
+import { WalletConstructorArgs } from "../types"
+import { UserWallet } from "../userWallet"
+import { btc2sat } from "../utils"
+import { HedgingStrategies, HedgingStrategy } from "./HedgingStrategyTypes"
+import { createHedgingStrategy } from "./HedgingStrategyFactory"
+
+const activeStrategy = HedgingStrategies.FtxPerpetualSwap
+// const activeStrategy = HedgingStrategies.OkexPerpetualSwap
+
+export class DealerWallet extends OnChainMixin(UserWallet) {
+  strategy: HedgingStrategy
+
+  constructor({ user, logger }: WalletConstructorArgs) {
+    super({ user, logger })
+    this.strategy = createHedgingStrategy(activeStrategy, logger)
```

Comment:
when passing several arguments to a function, I would typically recommend using an object. ie: 
`createHedgingStrategy({activeStrategy, logger})`. the main advantage is that you don't have to remember the order of the inputs if you use an object